src/index.js

```diff
@@ -12,15 +13,22 @@ const STATE_MAX = 2;
 const keys = ['x', 'y', 'width', 'mu'];
 /**
  * Fits a set of points to the sum of a set of bell functions.
- * @param {Object} data - An object containing the x and y data to be fitted.
- * @param {Array} peakList - A list of initial parameters to be optimized. e.g. coming from a peak picking [{x, y, width}].
- * @param {Object} [options = {}]
- * @param {String} [options.kind = 'gaussian'] - kind of shape used to fitting, lorentzian, gaussian and pseudovoigt are supported.
- * @param {Object} [options.lmOptions = {}] - options of ml-levenberg-marquardt optimization package.
- * @returns {Object} - A object with fitting error and the list of optimized parameters { parameters: [ {x, y, width} ], error } if the kind of shape is pseudoVoigt mu parameter is optimized.
+ * @param {object} data - An object containing the x and y data to be fitted.
+ * @param {array} peakList - A list of initial parameters to be optimized. e.g. coming from a peak picking [{x, y, width}].
+ * @param {object} [options = {}]
+ * @param {number|string} [options.kind = 'gaussian'] - kind of shape used to fitting, lorentzian, gaussian and pseudovoigt are supported.
+ * @param {object} [options.optimization = { kind: 'lm' }] - optimization options.
```

Comment:
Should be on another line
options.optimization={}
options.optimization.kind='lm'